invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -40,58 +45,75 @@ def requests_detail(request=None, default_query_config=None):
 
     topic_record = None
     is_draft = None
-    permissions = []
+    permissions = {}
 
-    is_topic_record_type = 'record' in request["topic"]
+    if community:
+        community_permissions = community.has_permissions_to(
+            ['update', 'read', 'search_requests', 'search_invites']
+        )
+        permissions.update(community_permissions)
+        community = community.to_dict()
 
-    if is_topic_record_type:
-        try:
-            topic_record = current_rdm_records_service.read_draft(
-                id_=request["topic"]["record"], identity=g.identity)
+    try:
+        topic_record = current_rdm_records_service.read_draft(
+            id_=request["topic"]["record"], identity=g.identity)
 
-        except NoResultFound:
-            topic_record = current_rdm_records_service.read(
-                id_=request["topic"]["record"], identity=g.identity)
+    except NoResultFound:
+        topic_record = current_rdm_records_service.read(
+            id_=request["topic"]["record"], identity=g.identity)
 
-        permissions = topic_record.has_permissions_to(
-            ['edit', 'new_version', 'manage',
-             'update_draft', 'read_files'])
-        topic_record = UIJSONSerializer().serialize_object_to_dict(
-            topic_record.data)
-        is_draft = topic_record["is_draft"]
-        request["topic"] = topic_record
+    permissions.update(topic_record.has_permissions_to(
+        ['edit', 'new_version', 'manage',
+         'update_draft', 'read_files'])
+    )
+    topic_record = UIJSONSerializer() \
+        .serialize_object_to_dict(topic_record.data)
+    is_draft = topic_record["is_draft"]
+    request["topic"] = topic_record
 
-        # end temporary block
+    # end temporary block
 
-        return render_template(
-            f"invenio_requests/community-submission/index.html",
-            # TODO: implement and use request UI serializer
-            request=request,
-            record=topic_record,
-            is_preview=True,
-            is_draft=is_draft,
-            permissions=permissions,
-            files=[],
-            default_query_config=default_query_config
-        )
+    return render_template(
+        f"invenio_requests/community-submission/index.html",
+        # TODO: implement and use request UI serializer
+        request=request,
+        record=topic_record,
+        is_preview=True,
+        is_draft=is_draft,
+        permissions=permissions,
+        files=[],
+        default_query_config=default_query_config,
+        custom_base_template=custom_base_template,
+        community=community,
+        user_avatar_url=avatar_url,
+        display_menu_only=display_menu_only,
+        types={
+            "organization": _("Organization"),
```

Comment:
This was changed by @alejandromumo... I believe that we should not hardcode it anymore.

Reply:
`types` is implemented as a vocabulary related to communities in https://github.com/inveniosoftware/invenio-communities/pull/531 , which is still unreleased. 

After it releases, the community has a `type` relation and there's no need to hard-code types anymore. Therefore passing `types` is not needed.  If you want to render a community's type, simply access `community.type` in the template.

Reply:
I just moved this code to split it, so it is a bit side scope from this PR, and I was not aware of progress with types. Since the communities types is it still not released, is it OK with everyone if it is addressed in a separate PR/issue ?

Reply:
I was checking the code of `invenio_requests/community-submission/index.html` and I don't find `types` value  being used there. I tried to check on that template's extends (`invenio_requests/details/index.html`) but I don't find the template. Am I missing something?

If that's the case and `types` is unused, maybe you don't need to inject `types` values in the context.  @kpsherva